src/main/resources/model/instance.json

```diff
@@ -111,16 +108,16 @@
           "showInResponse": true
         }
       }
+    },
+    "sort_contributors": {
+      "index": "lowercase_sort",
+      "populatedBy": "sortContributorsSetter"
+    },
+    "sort_title": {
+      "index": "lowercase_sort",
+      "populatedBy": "sortTitleSetter"
     }
   },
   "groups": { },
-  "indexMappings": {
```

Comment:
what was wrong with this implementation?

Reply:
I'm not sure but it doesn't work as expected, you can try it yourself.

My guess that sort_title group included all the properties that has `inventorySearchType: title`.